js/modules/k6/http/request.go

```diff
@@ -503,6 +503,12 @@ func (c *Client) Batch(reqsV ...goja.Value) (interface{}, error) {
 			err = e
 		}
 	}
+	for i, req := range batchReqs {
+		if req.Response != nil {
+			c.processResponse(req.Response, req.ParsedHTTPRequest.ResponseType)
+		}
+		batchReqs[i] = req
```

Comment:
I believe there is no need to re-assign the variable since in any case, `processResponse` works with the pointers
```suggestion
	for _, req := range batchReqs {
		if req.Response != nil {
			c.processResponse(req.Response, req.ParsedHTTPRequest.ResponseType)
		}
```